Dockerfile

```diff
@@ -6,7 +6,8 @@ COPY . /var/task/datadog-lambda-ruby
 WORKDIR /var/task/datadog-lambda-ruby
 RUN gem build datadog-lambda
 RUN gem install datadog-lambda --install-dir "/opt/ruby/gems/${runtime}"
-RUN gem install ddtrace --install-dir "/opt/ruby/gems/${runtime}"
+# TODO remove specific version when 0.48 is fixed
```

Comment:
Let's rephrase this comment -- we should note that 0.48 has a bug (can we link to a GitHub issue for tracking?) but we should probably keep a specific version here indefinitely so that the build is more deterministic.

Reply:
✅